board/nrf52832_thingy_52/board.h

```diff
@@ -0,0 +1,92 @@
+/* Copyright 2020 Wirepas Ltd. All Rights Reserved.
+ *
+ * See file LICENSE.txt for full license details.
+ *
+ */
```

Comment:
```suggestion
/* Copyright (c) 2023 grandcentrix GmbH
 *
 * SPDX-License-Identifier: MIT
 */
```

We probably don't want to put our code under the wirepas license so i guess its best to use a maximum permissive license like MIT.
Please add the MIT license to the repo as well, as `LICENSE-MIT` then it should be discoverable by github according to https://github.blog/changelog/2022-05-26-easily-discover-and-navigate-to-multiple-licenses-in-repositories/

Reply:
Done